### PR TITLE
[Content] Add docs for the new lock dims  of the roll button

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
# References and relevant issues
Depends on https://github.com/napari/napari/pull/5986

# Description
This PR adds basic documentation for the new feature added by @Chris-N-K https://github.com/napari/napari/pull/5986 to the Viewer tutorial in the `Roll dimensions` section.

### CODE REVIEW

**Comment by psobolewskiPhD** [comment]
Do y'all think this needs a graphic:
image

I think it's pretty self explanatory, but a visual can help...
(I made a suggestion to the PR to reduce the padding to make it less bloated. Also would be nice for the widget to show 3 dims if they exist, so this may change still.)

**Comment by melonora** [comment]
I think a visual can help as people directly know what they should see.

**Review by melonora — APPROVED** [review]
I do agree the visual could help so people know what they should expect to see. Other than that I think the text is good:)

**Comment by psobolewskiPhD** [comment]
@melissawm Once the final widget is merged, to add a graphic do I just put it in:
https://github.com/napari/docs/tree/main/docs/tutorials/assets/tutorials
and link, then commit and I'm good to go? there is no LFS or anything anymore, right?
(I'm basing off just above: `image: 3D_button` )

**Comment by melissawm** [comment]
Argh I missed this - yes, that's right, just push the image file to the repo :)

File: docs/tutorials/fundamentals/viewer.md [hunk 1]
```diff
@@ -409,7 +409,14 @@ viewer.camera.perspective = 45
...
 
-The third button rolls the dimensions that are currently displayed in the viewer. For example if you have a `ZYX` volume and are looking at the `YX` slice, this will then show you the `ZY` slice. You can also right-click this button to re-order the dimensions by drag-and-drop.
+The third button rolls the dimensions that are currently displayed in the viewer. 
+For example if you have a `ZYX` volume and are looking at the `YX` slice, this 
+will then show you the `ZY` slice. You can also right-click this button to pop-up 
+a widget that allows you to re-order the dimensions by drag-and-drop or lock a 
+dimension, by clicking on the padlock icon. Locking prevents a dimension from 
+being rolled (reordered). This can be particularly useful, for example, with a 
+`3D+time` dataset where you may want to fix the time dimension, while being able 
+to roll through the spatial dimensions.
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/viewer.md]:
```suggestion
to roll through the spatial dimensions.

```
Just making a null commit

**Comment by Chris-N-K** [comment]
Thanks a lot to all of you for providing the doc part to my PR :)

**Comment by psobolewskiPhD** [comment]
Grrr. still not right -- the image is huge. I'll try again tomorrow 😡

**Comment by psobolewskiPhD** [comment]
For future GitHub commit archeologists:
From:
https://myst-parser.readthedocs.io/en/latest/syntax/images_and_figures.html
> width : [length](https://myst-parser.readthedocs.io/en/latest/syntax/glossary.html#units-length) or [percentage](https://myst-parser.readthedocs.io/en/latest/syntax/glossary.html#units-percentage) of the current line width
The width of the image. Used to reserve space or scale the image horizontally. As with “height” above, when the “scale” option is also specified, they are combined.

**Comment by psobolewskiPhD** [comment]
@Chris-N-K 
I think this is the final form:
image
Available as html here:
https://output.circle-artifacts.com/output/job/9de6d541-d843-4cd8-9bc4-c2843e55d61b/artifacts/0/docs/docs/_build/html/tutorials/fundamentals/viewer.html#roll-dimensions

Is it Ok with you? if so, I'll mark ready to merge.
❤️

**Comment by Chris-N-K** [comment]
@psobolewskiPhD yes that's perfect :)
Thanks a lot and sorry for the wait I'm on vacation.

**Pull request merged**